modeller/chapter.md

```diff
@@ -216,12 +223,12 @@ Finally, `glFlush` signals to the graphics driver that we are ready for the buff
 ### What to Render: The Scene
 Now that we've initialized the rendering pipeline to handle drawing in the world coordinate space, what are we going to render? Recall that our goal is 
 to have a design consisting of 3D models. We need a data structure to contain the design, and we need use this data structure to render the design.
-Notice above that we call `self.scene.render()` from the Viewer's render loop. What exactly is the scene?
+Notice above that we call `self.scene.render()` from the viewer's render loop. What exactly is the scene?
```

Comment:
Changed "viewer" to lowercase where it isn't referred to directly by name, similar to how you would capitalize mom: "Is Mom coming today?", versus "Is your mom coming today?"